### PR TITLE
Address flaky update tests

### PR DESCRIPTION
**Pull request by ggediminass**

### CODE REVIEW

File: test/qa/lib/meshnet.py [hunk 1]
```diff
@@ -666,3 +667,28 @@ def is_connect_successful(output:str, peer_hostname: str):
...
+    """
+
+    for refresh_count in range(retry):
+        local_peer_list = sh.nordvpn.mesh.peer.list(_tty_out=False)
+        remote_peer_list = ssh_client.exec_command("nordvpn mesh peer list")
+
+        if "Status: connected" in local_peer_list and \
+            "Status: connected" in remote_peer_list:
+            logging.log(f"peer list refresh count: {refresh_count}")
+            break
```
Comment (mariusSincovici) [inline test/qa/lib/meshnet.py:689]: [small] I think if this is replaced with `return` +  `logging.log(f"===` is moved outside of `for`. then you don't need the next `if` :smile:

File: test/qa/lib/meshnet.py [hunk 2]
```diff
@@ -666,3 +667,28 @@ def is_connect_successful(output:str, peer_hostname: str):
...
+    Verifies if local and remote NordVPN mesh peers see each other as connected in peer list.
+
+    Args:
+        ssh_client (ssh.Ssh): SSH client to execute commands on the remote system.
+
+    Raises:
+        pytest.fail: If peers are not connected after `retry` attempts.
+    """
+
+    for refresh_count in range(retry):
```
Comment (mariusSincovici) [inline test/qa/lib/meshnet.py:682]: Shouldn't there be a delay not to have issues with the API?

**Review by mariusSincovici — APPROVED** [review]

File: test/qa/lib/meshnet.py (hunk 2 above)
Comment (ggediminass) [reply test/qa/lib/meshnet.py:682]: Good point, I did not think about this. Would you agree, that a delay of a second or two should be enough?

Comment (mariusSincovici) [reply test/qa/lib/meshnet.py:682]: yeah, 1-2 seconds should be fine

File: test/qa/lib/meshnet.py (hunk 1 above)
Comment (ggediminass) [reply test/qa/lib/meshnet.py:689]: My preference is to print peer lists only if failure occurs :smile:

**Review by devzbysiu — APPROVED** [review]

**Pull request merged**